subscription.go

```diff
@@ -48,11 +48,17 @@ type SubscriptionInput struct {
 	Name               string      `json:"name"`
 }
 
+type SubscriptionTerminateInput struct {
+	ExternalSubscriptionID string `json:"external_subscription_id,omitempty"`
+	Status								 string `json:"status,omitempty"`
```

Comment:
Sounds like we have a small indentation issue :)

Reply:
Yep! Fixed. I also reverted the attribute name change, we'll do that later